action-api/src/main/java/com/powsybl/action/DanglingLineAction.java

```diff
@@ -33,7 +31,7 @@ public class DanglingLineAction extends AbstractLoadAction {
      */
     DanglingLineAction(String id, String danglingLineId, boolean relativeValue, Double activePowerValue, Double reactivePowerValue) {
         super(id, relativeValue, activePowerValue, reactivePowerValue);
-        this.danglingLineId = Objects.requireNonNull(danglingLineId);
+        this.danglingLineId = danglingLineId;
```

Comment:
Why don't you keep danglingLineId required?
In the build() method, you should throw an exception if it's not set but the builder is here to allow us to have a temporary object without every required parameters, before building the action object with all required ones.